<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginMenu</class>
 <widget class="QMainWindow" name="loginMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="superwidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>9</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login1">
     <widget class="QGroupBox" name="loginWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>771</width>
        <height>381</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="entryDetails">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>60</y>
         <width>671</width>
         <height>141</height>
        </rect>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLineEdit" name="usernameEntry">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>521</width>
          <height>24</height>
         </rect>
        </property>
        <property name="placeholderText">
         <string>Input Your Username Here</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="passwordEntry">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>521</width>
          <height>24</height>
         </rect>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Input Your Password Here</string>
        </property>
       </widget>
       <widget class="QLabel" name="usernameLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="passwordLabel">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>79</y>
          <width>111</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="ErrorMessageHandler">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>310</y>
         <width>771</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
         <fontweight>Medium</fontweight>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>UpArrowCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 4);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>10</y>
         <width>151</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Kally's POS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="login">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>250</y>
         <width>81</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: bold 18px;</string>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="option1">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>691</width>
        <height>421</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="createOrderOptionButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Create Order</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewOrderOptionButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>View orders</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewManagementOptionButton">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Management</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="option1Title">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Kally's POS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="userIDLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:700;&quot;&gt;Kally's POS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="backFromOption1">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>490</y>
        <width>331</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QDockWidget" name="viewOrderDockWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>-20</y>
      <width>471</width>
      <height>401</height>
     </rect>
    </property>
    <property name="floating">
     <bool>true</bool>
    </property>
    <property name="features">
     <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetFloatable</set>
    </property>
    <widget class="QWidget" name="viewOrderDockWidgetContent">
     <widget class="QStackedWidget" name="viewOrderContent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>431</width>
        <height>371</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="orderList">
       <widget class="QListWidget" name="viewOrdersDisplayOrders">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>110</y>
          <width>256</width>
          <height>192</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>birds</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>arachnids</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>zombies</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>someone</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QWidget" name="orderInformation"/>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
